HelpSource/Classes/UnitTest.schelp

```diff
@@ -26,6 +26,19 @@ code::
 UnitTest.gui
 ::
 
+METHOD:: run
+Run all methods whose names begin with code::test_::.
```

Comment:
I find this wording a bit confusing.  Maybe:
"Run the receiving test class. Each of its class functions whose names begin with code::test_:: will be evaluated in turn."

Reply:
That's interesting, I'm struggling to understand how it could be even slightly confusing, but willing to accept that it is for other people, and certainly willing to work towards something better :-)  What aspect do you find confusing exactly?

>Run the receiving test class.

TBH I find that more confusing.  There is no single well-understood meaning for running a class.  Only methods can be run.

> Each of its class functions whose names begin with code::test_:: will be evaluated in turn.

I'm slightly uncomfortable with the use of the word "functions", since I thought the universally accepted name for these was "methods"?  IIUC functions are [defined and documented as a separate thing](http://doc.sccode.org/Reference/Functions.html) to methods.  Additionally, they're not class methods but instance methods.

And actually this highlights that neither my version nor yours really explains what's going on, which is that the `test_*` methods are not simply  run from this class method - after all a class method cannot directly call an instance method; something has to be instantiated first.  What actually happens is that an instance of the class is created for each `test_*` method, and then the framework is run for that test.  Although that is explained elsewhere in the document already, so perhaps this should just refer to that.

Reply:
I think of these methods more as functions because they are run inside a `Routine`. A routine runs a function, it does not call a method. I agree that my suggestion is not ideal.

Reply:
>'run' refers to test, not method. And indeed, the sentence doesn't explain the usage. So patrick's intuition is maybe this.

Yes, that sums up my gut feelings about this wording. However, I'm fine with it if @telephon is fine with it.